content/_data/authors/olblak.adoc

```diff
@@ -2,4 +2,8 @@
 name: "Olivier Vernin"
 twitter: 0lblak
 github: olblak
+irc: olblak
 ---
+Olivier is the Jenkins infrastructure officer and Senior Operations Engineer at CloudBees.
+As a regular contributor to the Jenkins infrastructure projects, he works on a wide range of tasks from services reliability to applications maintenance.
+Olivier is an infra coder who only believes in Git the mighty.
```

Comment:
I agree that it is simpler for everyone if the last line is removed.  Thanks for updating your bio!
```suggestion
```